index.js

```diff
@@ -0,0 +1,106 @@
+function ReadElement(selector) {
+  return document.querySelector(selector);
+}
+
+const menu = ReadElement('#menu-icon');
+const nav = ReadElement('.mobile-nav');
+const closeNav = ReadElement('.close-button');
+
+function AddMenu(selected, modalPart, opener) {
+  return selected.addEventListener('click', () => {
+    modalPart.classList.add(opener);
+    modalPart.style.top = '0px';
+  });
+}
```

Comment:
- [ ] As I notice that your mobile menu is not working please fix this issue and then you are good to go.